modules/cloudsql-instance/README.md

```diff
@@ -93,6 +93,59 @@ module "db" {
 }
 # tftest modules=1 resources=6
 ```
+
+### CMEK encryption pippo
+```hcl
+
+module "project" {
+  source          = "./modules/project"
+  billing_account = var.billing_account_id
+  parent          = var.organization_id
+  name            = "my-db-project"
+  services = [
+    "servicenetworking.googleapis.com"
+  ]
+}
+
+resource "google_project_service_identity" "jit_si" {
+  provider   = google-beta
+  project    = module.project.project_id
+  service    = "sqladmin.googleapis.com"
```

Comment:
You're already adding `sqladmin` to  modules/project/service-accounts.tf. Just adding `sqladmin.googleapis.com` to the project's services should be enough to trigger this.

Reply:
See #626